### ISSUE TITLE
[Bug Report] [el-button] [accessibility WCAG section508] 

### ISSUE THREAD

**Issue by hugo-valcourt** [ElemeFE/element#22408]
### Element UI version
2.15.13

### OS/Browsers version
MacOs Ventura 13.2.1 (22D68) / Chrome 110.0.5481.177 (arm64)

### Vue version
2.7.14

### Reproduction Link
https://element.eleme.io/#/en-US/component/button

### Steps to reproduce
Go on https://element.eleme.io/#/en-US/component/button
Open Google Lighthouse in dev tool, run Accessibility test

### What is Expected?
No Accessibility error.

### What is actually happening?
Buttons do not have an accessible name
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.
https://dequeuniversity.com/rules/axe/4.4/button-name?utm_source=lighthouse&utm_medium=devtools
Failing Elements
button.el-button

*****

Suggestions : could provide default title and/or aria-label ? With override through props ?

**Comment by webvs2** [comment]
Better accessibility is a harmonious operation, perhaps we can focus on here
